tutorials/integrate_3rd_libs_into_pipeline/README.md

```diff
@@ -0,0 +1,234 @@
+# Best Practices to integrate external libraries into Holoscan pipelines
```

Comment:
I like the rewording from "third party" to "external", as many of the libraries discussed here are developed in part by NVIDIA.

Could you please rename the project folder to match the new title? I.e., rename from `integrate_3rd_libs_into_pipeline` to `integrate_external_libs_into_pipeline`.